projects/canopy/src/styles/spacing.scss

```diff
@@ -46,16 +48,18 @@ $spacing-list: (
   }
 }
 
-@each $val in $spacing-list {
-  .lg-margin--#{$val} {
-    margin: space($val) !important;
-  }
+// Default margin / padding
+// @each $val in $spacing-list {
```

Comment:
I think these might need to stay 🤔. Would it create much of an issue?
I think we said that these classes would come under semantic versioning.
https://github.com/Legal-and-General/canopy/blob/master/docs/BREAKING_CHANGES.md

Reply:
Ah good point, I wasn't sure about this, and was partly commenting them out here to check that the dynamic style service was definitely working. 

I've opened a discussion to cover this: https://github.com/Legal-and-General/canopy/discussions/509